.github/workflows/integration.yml

```
@@ -85,7 +85,10 @@ jobs:
         env:
           ARCHERY_DOCKER_USER: ${{ secrets.DOCKERHUB_USER }}
           ARCHERY_DOCKER_PASSWORD: ${{ secrets.DOCKERHUB_TOKEN }}
-        run: archery docker run -e ARCHERY_INTEGRATION_WITH_RUST=1 conda-integration
+        run: >
+          archery docker run -e ARCHERY_INTEGRATION_WITH_RUST=1 -e
+          ARCHERY_DEFAULT_BRANCH=${{ github.event.repository.default_branch }}
+          conda-integration
```

How about indenting/formatting for easy to read?

```suggestion
          archery docker run
             -e ARCHERY_DEFAULT_BRANCH=${{ github.event.repository.default_branch }}
             -e ARCHERY_INTEGRATION_WITH_RUST=1
             conda-integration
```